esp/bindings/http/client/httpclient.cpp

```diff
@@ -189,7 +189,7 @@ CHttpClient::~CHttpClient()
         }
         else if (m_persistentHandler && !m_disableKeepAlive && m_persistable)
         {
-            m_persistentHandler->add(m_socket, &m_ep);
+            m_persistentHandler->add(m_socket, &m_ep, strieq(m_protocol.get(), "HTTPS")?PersistentProtocol::ProtoTls:PersistentProtocol::ProtoDefault);
```

Comment:
Unless ProtoDefault is always HTTP, then is there no way of explicitly creating an HTTP entry?  I.e. if ProtoTls and ProtoDefault are both HTTPS.

Reply:
For https ProtoTls should always be used. But I can see how "Default" could be misleading. Let me change it to "Regular".

Reply:
Ok.  Or TCP.

Reply:
TCP does sound better, let me change to that.